arbitrum-docs/stylus/how-tos/debugging-stylus-tx.mdx

```diff
@@ -107,7 +107,7 @@ export TX_HASH=0x18b241841fa0a59e02d3c6d693750ff0080ad792204aac7e5d4ce9e20c46683
 And replay the transaction:
 
 ```sh
-cargo stylus replay --tx=$TX_HASH --endpoint=$RPC_URL
+cargo stylus replay --tx=$TX_HASH --endpoint=$RPC_URL --use-native-tracer
 ```
```

Comment:
```sh
Options:

--tx: Specifies the transaction hash to replay.
--endpoint: Specifies the RPC endpoint for fetching transaction data.
--use-native-tracer: Utilizes the native Stylus tracer for enhanced debugging. This tracer is specifically designed for the Stylus runtime and improves debugging accuracy.
```



```

Reply:
:::caution Supported RPC Endpoints Only

The `--use-native-tracer` flag requires access to an RPC endpoint with Stylus tracing support, See the list of providers that support Stylus tracing [here](https://docs.arbitrum.io/for-devs/dev-tools-and-resources/chain-info#third-party-rpc-providers).

:::